kyaml/kio/byteio_reader_test.go

```diff
@@ -805,3 +805,161 @@ items:
 		})
 	}
 }
+
+// TestByteReader_AddSeqIndentAnnotation tests if the internal.config.kubernetes.io/seqindent
+// annotation is added to resources appropriately, the expectedOutput indentation may not
+// match with the annotation as it is not using byteio_writer, this test will only verify
+// byteio_reader behavior to add annotation
+func TestByteReader_AddSeqIndentAnnotation(t *testing.T) {
+	type testCase struct {
+		name                  string
+		err                   string
+		input                 string
+		expectedOutput        string
```

Comment:
This test intend to test the behavior that that the ByteReader is putting the correct annotations (relevant to SeqIndent) in the read YAML but having the entire YAML content in `expectedOutput` confused me a bit where I started to expect the indentation style in the `expectedOutput` to match the one in the annotation (which is really not relevant).
So if we can narrow the expected content to be just the annotations in the readYAML, we can get rid of the confusion.

Reply:
Good point. We just need to check for the annotation value added. So changed to `expectectedAnnoValue`